src/app/settings.py

```diff
@@ -310,17 +328,17 @@ def show_toolbar(request):
     }
 }
 
-if ENVIRONMENT in ("dev", "prod"):
-    LOGGING["handlers"]["watchtower"] = {
-        'level': DEBUG_LEVEL,
-        'class': 'watchtower.CloudWatchLogHandler',
-        'formatter': 'file',
-        'log_group': '{}-mermaid-api'.format(ENVIRONMENT),
-        'use_queues': True,
-        'boto3_client': boto3_client
-    }
-    LOGGING["loggers"][""]["handlers"].append("watchtower")
-
+# NOTE not required in ECS
```

Comment:
Great -- so can be removed. But let's talk about how logging will be handled.

Reply:
Yes, this reminds me... If we're removing these things from the API, it might break things in the existing API once we merge... We'll need to think about how this will work to allow both new/old APIs to run within the same codebase. ie: feature flags?